README.md

```diff
@@ -25,13 +25,17 @@ pip install --user -r requirements.txt
 On server (PC which has controller connected to it.) run:
 
 ```bash
-./controller/server.py --public
+DEBUG_LEVEL=INFO ./controller/server.py --public
```

Comment:
since we have the note below, these commands should not have DEBUG_LEVEL prefixed already